src/Controllers/expenses.controller.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+const service = require('../services/expenses.services');
+const userService = require('../services/users.services');
+
+const getExpenses = (req, res) => {
+  const {
+    userId,
+    from,
+    to,
+  } = req.query;
+
+  const normalizedUrl = new URL('http://localhost:3000/' + req.url);
+  const categories = normalizedUrl.searchParams.getAll('categories');
```

Comment:
```suggestion
    to,
    categories,
  } = req.query;
```

Reply:
when i did it this way, 4 tests were failing, but I see now
it was because of  ` if (categories.length) `
😀😀😀😀 (😭💀)